dask_cudf/io/tests/test_csv.py

```diff
@@ -24,6 +25,29 @@ def test_read_csv_w_bytes(tmp_path):
     df.to_csv(tmp_path / "data-*.csv", index=False)
 
     df2 = dask_cudf.read_csv(tmp_path / "*.csv", chunksize="50 B")
+    assert df2.npartitions is 3
+
+    result = df2.compute().to_pandas()
+    expected = df.compute()
+    dd.assert_eq(result, expected, check_index=False)
+
+
+
+def test_read_csv_compression(tmp_path):
+    df = dask.datasets.timeseries(dtypes={"x": int, "y": int}, freq="120s").reset_index(
+        drop=True
+    )
+    df = pd.DataFrame(dict(x=np.arange(20), y=np.arange(20)))
+    df = dd.from_pandas(df, npartitions=1)
+    df.to_csv(tmp_path / "data-*.csv", index=False)
```

Comment:
I think that you can add a `compression='gzip'` here if you like .  

It would also be good to ensure that we can actually read this data.

Reply:
fixed in last commit